Platforms/iOS/MTHKView.swift

```diff
@@ -103,9 +104,18 @@ extension MTHKView: MTKViewDelegate {
             break
         }
         let bounds = CGRect(origin: .zero, size: drawableSize)
-        let scaledImage: CIImage = displayImage
+        var scaledImage: CIImage = displayImage
             .transformed(by: CGAffineTransform(translationX: translationX, y: translationY))
             .transformed(by: CGAffineTransform(scaleX: scaleX, y: scaleY))
+
+        if isMirrored {
+            if #available(iOS 11.0, *) {
+                scaledImage = scaledImage.oriented(.upMirrored)
+            } else {
+                scaledImage = scaledImage.oriented(forExifOrientation: 2)
+            }
+        }
+
```

Comment:
The orientation needs to happen before the transformation. @shogo4405

Reply:
Hi @hambly, I am not familiar with the CGAffineTransform part, can you explain more about why does the orientation needs to happen before the transformation?
The appended PR and comments are welcome, too.

Reply:
Orientation is inherent to the media, the transformation is part of rendering for display, in particular the translation aspect.

Reply:
So you mean the results would be the same here, but we should usually do orientation before transformation for the process, correct?

Reply:
The results are not the same here, this PR did not correctly configure the mirrored front-facing camera, as the translation becomes incorrect upon mirroring.

See https://github.com/shogo4405/HaishinKit.swift/pull/675

Reply:
I am not sure how can the translation becomes incorrect upon mirroring...
May I ask how did the PR https://github.com/shogo4405/HaishinKit.swift/pull/675/files being closed? Is it a correct PR that should be merged?

You can propose a PR to amend this one, or I will clarify it with and give a PR based on your suggestion.